appchain/src/mock.rs

```diff
@@ -282,6 +282,11 @@ impl pallet_uniques::Config for Test {
 	type Locker = ();
 }
 
+parameter_types! {
+	pub const NativeTokenDecimals: u128 = 1000_000_000_000_000_000;
```

Comment:
It would be nice to be able to read the number from chainspec. And the number should be 18 in this case.